src/tests/ftest/ior/hard_rebuild.py

```diff
@@ -54,11 +54,13 @@ def test_ec_ior_hard_online_rebuild(self):
         # Do not change the stoneWallingStatusFile during read
         self.ior_cmd.sw_wearout.update(None)
 
+        # remove first ior_read_dataset() calling, as it will zero the stoneWallingStatusFile
+        # and then fail the following 2nd ior_read_dataset (details in DAOS-10342).
         # Disabled Online rebuild
-        self.set_online_rebuild = False
+        # self.set_online_rebuild = False
         # Read IOR data and verify for different EC object kill single server while IOR Read phase
         # is in progress.
-        self.ior_read_dataset()
+        # self.ior_read_dataset()
```

Comment:
This was fixed as part of https://github.com/daos-stack/daos/pull/7139 so it should not be  zero the stoneWallingStatusFile unless something changed in the way IOR works

Reply:
please check comment on the ticket, that fix depend on 150S stop the reader, but within 150S it possibly goes beyond EOF.